list-ops/list_ops_test.py

```diff
@@ -0,0 +1,98 @@
+import unittest
+import math
+
+from list_ops import map_src, length_src, reduce_src
+
+
+class ListOpsTest(unittest.TestCase):
+
+    # necessary functions to test map_src
+    def square(self, x):
+        return x**2
+
+    def cube(self, x):
+        return x**3
+
+    def plus_one(self, x):
+        return x + 1
+
+    def minus_one(self, x):
+        return x - 1
+
+    # tests for map
+    def test_map_square(self):
+        self.assertEqual([1, 4, 9, 16, 25, 36, 49, 64, 81, 100], map_src(
+            self.square, [-1, 2, -3, 4, -5, 6, -7, 8, -9, 10]))
```

Comment:
This will probably be easier to read if the second argument is kept on a single line.